spec/support/page_objects/publish/course_confirmation.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative "../sections/errorlink"
+module PageObjects
```

Comment:
spacing here please

Reply:
done in https://github.com/DFE-Digital/teacher-training-api/pull/2525/commits/77827debc4ed2be38058f60183926ff5264ee34a